docs/tutorial/topics/custom-icons.rst

```diff
@@ -161,9 +161,15 @@ This reports the specific icon file (or files) that Briefcase is expecting.
 However, as we haven't provided the actual icon files, the install fails, and
 Briefcase falls back to a default value (the same "gray bee" icon).
 
+
+
+
```

Comment:
Reduce, reuse, recycle your electrons! We don't need all these extra blank lines.

```suggestion
```